CONTRIBUTING.md

```diff
@@ -74,6 +85,46 @@ branches that minor and patch releases are tagged. In some cases, it may
 be necessary to open PRs for bugfixes directly against stable branches, but
 this should generally not be the case.
 
+### Support and guarantees
+
+Cluster API Provider Metal3 maintains the most recent release/releases for all supported API and contract versions. Support for this section refers to the ability to backport and release patch versions;
+[backport policy](#backporting) is defined above.
+
+- The API version is determined from the GroupVersion defined in the top-level `api/` package.
+- The EOL date of each API Version is determined from the last release available once a new API version is published.
+
+| API Version  | Supported Until                                                              |
```

Comment:
You don't need to draw literal tables in markdown, the renderer will handle that.

Reply:
This is nice to read in raw mode though :wink: some plugins anyway automate this.

Reply:
Keeping it.